src/app/pages/organizations/developers/developer/join.jsx

```diff
@@ -93,94 +132,108 @@ function ChplJoinDevelopers({ id }) {
-                     <TableRow key={developer.id}>
-                       <TableCell>{developer.developerCode}</TableCell>
-                       <TableCell>{developer.name}</TableCell>
-                       <TableCell>{developer.status.status}</TableCell>
-                       <TableCell>
-                         <Button
-                           onClick={() => removeDeveloper(developer)}
-                         >
-                           Remove
-                         </Button>
-                       </TableCell>
-                     </TableRow>
-                   ))}
-                 </TableBody>
-               </Table>
-             </TableContainer>
-           )}
-        </CardContent>
-      </Card>
-      <Card>
-        <CardHeader title={`Select Developers joining ${activeDeveloper.name}`} />
-        <CardContent>
-          { /* eslint-disable react/jsx-props-no-spreading */}
-          <Autocomplete
-            id="developers"
-            name="developers"
-            options={developers}
-            onChange={addDeveloper}
-            inputValue={developerValueToLoad}
-            onInputChange={(event, newValue) => {
-              setDeveloperValueToLoad(newValue);
-            }}
-            getOptionLabel={(item) => `${item.name} (${item.developerCode})`}
-            renderInput={(params) => <ChplTextField {...params} label={`Select Developers joining ${activeDeveloper.name}`} />}
-          />
-          { /* eslint-enable react/jsx-props-no-spreading */}
-          <TableContainer>
-            <Table>
-              <TableHead>
-                <TableRow>
-                  <TableCell>Code</TableCell>
-                  <TableCell>Name</TableCell>
-                  <TableCell>Status</TableCell>
-                  <TableCell><span className="sr-only">Action</span></TableCell>
-                </TableRow>
-              </TableHead>
-              <TableBody>
-                {developers.map((developer) => (
-                  <TableRow key={developer.id}>
-                    <TableCell>{developer.developerCode}</TableCell>
-                    <TableCell>{developer.name}</TableCell>
-                    <TableCell>{developer.status.status}</TableCell>
-                    <TableCell>
-                      <Button
-                        onClick={() => addDeveloper(undefined, developer)}
-                        disabled={!canAdd(developer)}
-                      >
-                        Add
-                      </Button>
-                    </TableCell>
-                  </TableRow>
-                ))}
-              </TableBody>
-            </Table>
-          </TableContainer>
-        </CardContent>
-      </Card>
+      <Box p={8} bgcolor="#fff">
+        <Typography variant="h1">
+          Join Developers
+        </Typography>
+      </Box>
+      <Container disableGutters maxWidth="xl">
+        <Box className={classes.boxContainer}>
```

Comment:
I've noticed a lot of "Box" elements in these PRs recently. Why use Boxes instead of DIVs? Is there a benefit I don't know about?

Reply:
you can inject css right into "box"s. Brian H showed me this. Just a quicker way instead of DIVs / classes